conformance/client/src/main/kotlin/com/connectrpc/conformance/client/adapt/BidiStreamClient.kt

```diff
@@ -44,20 +44,18 @@ abstract class BidiStreamClient<Req : MessageLite, Resp : MessageLite>(
      * @param Resp The response message type
      */
     interface BidiStream<Req : MessageLite, Resp : MessageLite> {
-        fun requests(): RequestStream<Req>
-        fun responses(): ResponseStream<Resp>
+        val requests: RequestStream<Req>
+        val responses: ResponseStream<Resp>
```

Comment:
I've changed a few things in these interfaces to be property getters instead of methods since they really are lightweight properties.